src/main/java/com/hrznstudio/titanium/plugin/PluginPhase.java

```diff
@@ -0,0 +1,12 @@
+package com.hrznstudio.titanium.plugin;
+
+public enum PluginPhase {
+    CONSTRUCTION,
+    CLIENT_SETUP,
+    COMMON_SETUP,
+    PRE_INIT,
```

Comment:
Because this way mods can execute code properly before all the module controller things without having race conditions or not exexuting code when you want to.